java/mtsj/core/src/main/java/com/devonfw/application/mtsj/usermanagement/dataaccess/api/repo/UserRepository.java

```diff
@@ -29,6 +39,10 @@
     if ((username != null) && !username.isEmpty()) {
       QueryUtil.get().whereString(query, $(alias.getUsername()), username, criteria.getUsernameOption());
     }
+    String password = criteria.getPassword();
+    if ((password != null) && !password.isEmpty()) {
+      QueryUtil.get().whereString(query, $(alias.getPassword()), password, criteria.getPasswordOption());
+    }
```

Comment:
...like my first comment above. Was that introduced via CobiGen regeneration or by a merge bug?
We can not seriously want to allow users to search for other users by their password...

Reply:
If CobiGen is causing all this and we have a thread of re-introducing such security problems again in the future we should even think about splitting the user table such that the password is not contained in User[(Entity|Eto)] anymore...

Reply:
Done